src/config/dark-sites.config

```diff
@@ -96,6 +96,7 @@ blog.aractus.com
 blog.counter-strike.net
 blog.dota2.com
 blog.ja-ke.tech
+blog.kyleggiero.me
```

Comment:
You don't have to add any subdomain. the domain itself is sufficient.

Reply:
Oh interesting! I was just following the lead of others in the file. Will push that, thank you

Reply:
Oh interesting! I was just following the lead of others in the file. Will push that, thank you

Reply:
@Gusted it seems that blog.kyleggiero.me is now being treated as if it's not on this list. What happened?